src/Layer/ElevationLayer.js

```diff
@@ -54,6 +61,8 @@ class ElevationLayer extends RasterLayer {
      */
     constructor(id, config = {}) {
         super(id, config);
+        this.zmin = config.clampValues?.min ?? config.zmin;
```

Comment:
What if `config.zmin` and `config.zmax` are not set either? Also, `config.zmin` and `config.zmax` are not documented and the same as the `clampValues` ?
Shouldn't it be something like: 
```
if (config.clampValues && config.clampValue.min) {
  this.zmin = config.clampValues.min;
}
```
or like
```
this.zmin = config.clampValues?.min ?? -Infinity;
```
?

Reply:
I kept config.zmin and config.zmax in dicussion with @mgermerie to limit break change if someone used them before (even if there were not really any documentation of these).
And when both are undefined, it normally not a problem as the default value (-Infinity and Infinity) will be taken. see RasterTile.js line 150
```
        this.zmin = layer.zmin ?? defaultEle.zmin;
        this.zmax = layer.zmax ?? defaultEle.zmax;
```

Reply:
About the documentation i added some line for it :
```
 * @property {number} noDataValue - Used to specify a **null** or **no data value** in the elevation terrain.
 * @property {number} zmin - Used to specify a minimum value under which the elevation terrain will be clamped.
 * @property {number} zmax - Used to specify a maximum value over which the elevation terrain will be clamped.
```
and
```
     * @param {Object} [config] - Optional configuration, all elements in it
     * will be merged as is in the layer. For example, if the configuration
     * contains three elements `name, protocol, extent`, these elements will be
     * available using `layer.name` or something else depending on the property
     * name.
     * @param {number} [config.noDataValue]   The value coding the noData in the data set
     * @param {Object} [config.clampValues] - Optional information for clamping
     * the elevation between a minimum and a maximum value
     * @param {number} [config.clampValues.min]   The minimum value to clamp the elevation
     * @param {number} [config.clampValues.max]   The maximum value to clamp the elevation
```

Reply:
Ok. In this case my opinion is that we should:
* not document them because we don't want users to use these but to use `config.clampValues.min` and `config.clampValues.max` (and we don't want to have two possibilities to configure the same way I think, it complicates the API)
* Add a deprecation warning for these config values

Reply:
I removed the documentation of the layer properties zmin, zmax and noDataValue.
And i added a deprecated warning if zmin and zmax are directly used.
```
        if (config.zmin || config.zmax) {
            console.warn('Config using zmin and zmax are deprecated, use {clampValues: {min, max}} structure.');
        }
```

Reply:
Ok sorry I think there was a mixup between the Layer properties documentation and the constructor documentation :sweat_smile: Could you please:
* In the layer: leave the doc about zmin, zmax and noDataValue since they are public properties
* In the constructor: leave it like it is currently (with clampValues and noDataValue documented)